mpt_trie/src/debug_tools/diff.rs

```diff
@@ -423,11 +423,11 @@ mod tests {
     fn depth_single_node_hash_diffs_work() {
```

Comment:
nit: can have the test return `TrieOpResult` and just use `?` instead of `assert!(_.is_ok())`.